http/swagger.yaml

```diff
@@ -51,3 +52,65 @@ paths:
           description: Message discarded due to invalid or missing content type.
         500:
           description: Unexpected server-side error occurred.
+definitions:
+  SenML:
+    type: object
+    properties:
+      bn:
+        type: string
+        description: Base Name
+      bt:
+        type: number
+        format: double
+        description: Base Time
+      bu:
+        type: number
+        format: double
+        description: Base Unit
+      bv:
+        type: number
+        format: double
+        description: Base Value
+      bs:
+        type: number
+        format: double
+        description: Base Sum
+      bver:
+        type: number
+        format: double
+        description: Version
+      n:
+        type: string
+        description: Name
+      u:
+        type: string
+        description: Unit  
+      v:
+        type: number
+        format: double
+        description: Value
+      vs:
+        type: string
+        description: String Value
+      vb:
+        type: boolean
+        description: Boolean Value
+      vd:
+        type: string
+        description: Data Value
+      s:
+        type: number
+        format: double
+        description: Value Sum
+      t:
+        type: number
+        format: double
+        description: Time
+      ut:
+        type: number
+        format: double
+        description: Update Time
+  array:
```

Comment:
I would call this one SenML

Reply:
There is SenML declaration just above this declaration.